pgjdbc/src/main/java/org/postgresql/jdbc/TimestampUtils.java

```diff
@@ -518,8 +518,24 @@ public OffsetTime toOffsetTimeBin(byte[] bytes) throws PSQLException {
 
   /**
    * Parse a string and return a OffsetDateTime representing its value.
+   * This function is here to retain compatibility with the previous version 42.3.3
+   * for converting timestamp and timestamptz for timetz use #toOffsetDateTime(timetz, false)
    *
-   * @param s The ISO formated date string to parse.
+   * @param s The ISO formatted date string to parse.
+   * @return null if s is null or a OffsetDateTime of the parsed string s.
+   * @throws SQLException if there is a problem parsing s.
+   * @deprecated in favour of #toOffsetDateTime(String s, boolean adaptToUTC)
```

Comment:
I think we suggest users to refrain from using `TimestampUtils` directly. We don't really want them to call `toOffsetDateTime(String s, boolean adaptToUTC)`.